### PR TITLE
Codacy Issue fixes for ha files

### PR DESCRIPTION
**Pull request by rahul-hatwar**
Signed-off-by: Rahul Hatwar <rahulchandrakant.hatwar@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by rahul-hatwar** [comment]
's3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_get_head_object.py::TestVersioningGetHeadObject::test_get_head_object_versioned_bucket_32731', 'test_id': 'TEST-32731', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_get_head_object.py::TestVersioningGetHeadObject::test_get_head_object_versioned_bucket_32729', 'test_id': 'TEST-32729', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_get_head_object.py::TestVersioningGetHeadObject::test_get_head_versioned_object_invalid_32727', 'test_id': 'TEST-32727', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_get_head_object.py::TestVersioningGetHeadObject::download_and_check', 'test_id': '', 'marks': []}, {'nodeid': 'tests/s3/versioning/test_versioning_put_object.py::TestVersioningPutObject::test_put_object_preexisting_32724', 'test_id': 'TEST-32724', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_put_object.py::TestVersioningPutObject::test_put_object_versioning_enabled_32728', 'test_id': 'TEST-32728', 'marks': ['s3_ops']}, {'nodeid': 'tests/s3/versioning/test_versioning_put_object.py::TestVersioningPutObject::test_put_object_versioning_suspended_32733', 'test_id': 'TEST-32733', 'marks': ['s3_ops']}, {'nodeid': 'tests/security/test_cortx_port_scanner_kubectl_svc.py::test_cortx_port_scanner_kubectl_svc', 'test_id': 'TEST-34217', 'marks': ['security']}, {'nodeid': 'tests/security/test_cortx_port_scanner_netstat.py::test_cortx_port_scanner_netstat', 'test_id': 'TEST-34218', 'marks': ['security']}] created at /root/rahul/cortx-test-1/log/te_meta.json
Successfully unmounted directory

=================================================== 2163 tests collected in 8.20s ====================================================

File: comptests/ha/test_restart_pod.py [hunk 1]
```diff
@@ -93,7 +93,8 @@ def teardown_method(self):
                                                            "deployment_backup":
                                                                self.deployment_backup})
             LOGGER.debug("Response: %s", resp)
-            assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} way")
+            assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} "
+                                              f"way")
```
Comment (ApurwaMohite) [inline comptests/ha/test_restart_pod.py:97]: You will get warning for this f"".

File: comptests/ha/test_restart_pod.py [hunk 2]
```diff
@@ -213,9 +214,10 @@ def test_datapod_status_online_after_delete_deployment(self):
         LOGGER.info("Deleting pod %s", pod_name)
         resp = self.node_master_list[0].delete_deployment(pod_name=pod_name)
         LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_false(resp[0], f"Failed to delete pod {pod_name} by deleting deployment"
-        " (unsafe)")
-        LOGGER.info("Step 1: Successfully shutdown/deleted pod %s by deleting deployment", pod_name)
+        assert_utils.assert_false(resp[0], f"Failed to delete pod {pod_name} by deleting"
+                                           f" deployment (unsafe)")
```
Comment (ApurwaMohite) [inline comptests/ha/test_restart_pod.py:218]: Same here.

File: comptests/ha/test_restart_pod.py [hunk 3]
```diff
@@ -347,15 +349,16 @@ def test_serverpod_status_online_after_delete_deployment(self):
...
+            self.node_master_list[0].get_all_pods(pod_prefix=common_const.SERVER_POD_NAME_PREFIX)
         pod_name = random.sample(pod_list, 1)[0]
         LOGGER.info("Deleting pod %s", pod_name)
         resp = self.node_master_list[0].delete_deployment(pod_name=pod_name)
         LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_false(resp[0], f"Failed to delete pod {pod_name} by deleting deployment"
-        " (unsafe)")
-        LOGGER.info("Step 1: Successfully shutdown/deleted pod %s by deleting deployment", pod_name)
+        assert_utils.assert_false(resp[0], f"Failed to delete pod {pod_name} by deleting "
+                                           f"deployment (unsafe)")
```
Comment (ApurwaMohite) [inline comptests/ha/test_restart_pod.py:359]: Same here.

**Review by ApurwaMohite — APPROVED** [review]

**Review by sonalk0209 — APPROVED** [review]

**Review by ankita-93 — APPROVED** [review]
lgtm

**Review by seagate-sarang-sawant — APPROVED** [review]

**Pull request merged**